jekyll/_cci2/server/Installation/phase-2-core-services.adoc

```diff
@@ -232,30 +232,78 @@ You will need to update your DNS records to the new loadbalancer once you have r
 
 [#step-four-github-integration]
 == Step four: GitHub integration
+To configure GitHub with CircleCI, there are two options for providing credentials to the deployment.
 
-You created your Github OAuth application in the prerequisite phase. Add the client ID and secret.
+[#github]
+=== GitHub
+These instructions are for the non-enterprise version of GitHub.
 
+* *Option 1 - Create the secret yourself*
++
+[source,shell]
+----
+kubectl create secret generic github-secret \
+  --from-literal=clientId=<github-client-id> \
+  --from-literal=clientSecret=<github-client-secret> 
```

Comment:
I think it makes sense to be consistent with the naming, could go the other way too, i have not strong opinion on that

```suggestion
  --from-literal=clientId=<client-id> \
  --from-literal=clientSecret=<client-secret> 
```